.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "submodules\\AllReady"]
+	path = submodules\\AllReady
```

Comment:
I think you should use forward slash: `submodules/AllReady`. Otherwise I'm not sure if the submodule would work on git/Linux.

Reply:
@jorive thanks! that was it!